ansible/group_vars/all

```diff
@@ -102,6 +102,10 @@ dashd_minimumdifficultyblocks: 4032
 # You can provide your own seed node in network config. May be useful if you want to connect to another network
 dashd_seednode: '{{ hostvars[groups.seed_nodes[0]]["public_ip"] if groups.seed_nodes else none }}'
 
+# This allows us to identify a wallet node so we can add the mixing stuff in the dashd role
```

Comment:
Are you confusing wallet nodes and mixing nodes here?

Reply:
No, this was when we were using the wallet nodes and mixing nodes weren't a thing.... that approach completely failed. I'll update it